docs/authentication/saml/authentication-flows.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Authentication flows
+description: Learn about the SAML SSO Authentication flows.
+---
+
+# Authentication flows
+
+The SAML protocol supports two different methods in order to start an SSO flow: SP-initiated and IdP-initiated. Below we will explore
+how each of them works and the differences between the two.
+
+## SP-initiated flow
+In an SP-initiated flow, the user is starting the authentication flow from your application (SP), by providing the email address. Then they are being redirected to the SAML provider (IdP) where
```

Comment:
```suggestion
In an SP-initiated (service-provider-initiated) flow:

- The user starts the authentication flow from your application by providing the email address.
- The user is redirected to the SAML provider (IdP), where they must authenticate themselves.
```

Reply:
As i am aware the term that SAML is using is `SP-initiated` all the time, so it guess it's fine to leave it as is?